src/script/search/SearchRepository.ts

```diff
@@ -177,6 +167,11 @@ export class SearchRepository {
     }, 0);
   }
 
+  private async getContacts(query: string, numberOfRequestedUser: number, domain?: string): Promise<SearchResult> {
+    const request = await this.apiClient.api.user.getSearchContacts(query, numberOfRequestedUser, domain);
+    return request.response;
+  }
```

Comment:
This was the only method in the `SearchService`. Did not really make sense to have it there